client/src/components/starRatings.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import StarRatings from 'react-star-ratings';
+
+const StarRating = ({ rating, count }) => (
+  <div>
+    <StarRatings
+      rating={rating}
+      starRatedColor="#F5B895"
+      starEmptyColor="#B5C7D3"
+      numberOfStars={5}
+      starDimension="25px"
+      starSpacing="1px"
+      name="rating"
+    />
+    <div className="stars-review-link stars-spacing">
+      <a href="https://github.com/callback-kids/ratings-and-reviews">
+        View&nbsp;
+        {count}
+        &nbsp;Reviews
```

Comment:
If these `&nbsp;` tags are for creating spaces in the text, possibly turn the entire section into a template literal to simplify the code a bit. Could be wrong here

Reply:
that's a good idea, the eslinter suggested these tags for screen readers, I initially started with a template literal and it doesn't like trailing white spaces, even inside the backticks.